SharingIsCaring.rb

```diff
@@ -0,0 +1,35 @@
+=begin
+Given an array of numbers, create a function that removes 25% from every number in the array except the smallest number, and adds the total amount removed to the smallest number.
+
+Examples
+show_the_love([4, 1, 4]) ➞ [3, 3, 3]
+
+show_the_love([16, 10, 8]) ➞ [12, 7.5, 14.5]
+
+show_the_love([2, 100]) ➞ [27, 75]
+Notes
+There will only be one smallest number in a given array.
+=end
+def love(removed, numbers)
+  numbers.map{ |number|
```

Comment:
```suggestion
  numbers.map do |number|
  
  end
```

use this syntax for multi line blocks. Does the same thing but its better ruby syntax than the {}

Reply:
Okay.